q_it_argument_error.rb

```diff
@@ -0,0 +1,8 @@
+class QItArgumentError < StandardError
+  
+  def initialize obj
+    puts "Error in #{obj}"
```

Comment:
Not sure the reason behind the `puts`

Reply:
Wanted to communicate which object is giving an error.

Reply:
Why `puts`?
There is also no need for initialize unless you are trying to add something extra to the `QItArgumentError` class. Otherwise something like below should work.
```
class QItArgumentError < StandardError ; end

raise QItArgumentErrror.new("Custom Error")
```
Try the above in `irb`.